CMakeLists.txt

```diff
@@ -117,14 +117,15 @@ set(CMAKE_SHARED_LINKER_FLAGS_DEBUG "${CMAKE_SHARED_LINKER_FLAGS_DEBUG} -O0")
 set(CMAKE_EXE_LINKER_FLAGS_DEBUG "${CMAKE_EXE_LINKER_FLAGS_DEBUG} -O0")
 
 # Dependencies
+set(DEFAULT_LIB_DIRS $ENV{HOME}/local /opt/local /usr/local /usr)
 find_package(Numa QUIET)
 find_package(Tbb REQUIRED)
 find_package(Readline REQUIRED)
 find_package(Curses REQUIRED)
-find_package(Sqlite3 REQUIRED)
-find_package(Boost REQUIRED COMPONENTS container system date_time)
+find_package(SQLite3 REQUIRED)
+find_package(Boost REQUIRED COMPONENTS container date_time)
 
-add_definitions(-DBOOST_THREAD_VERSION=5)
```

Comment:
Why can we remove the boost thread version? Or better question: why did we need it in the first place?

Reply:
I started with the latter question and haven't found an answer yet. That's why I removed it.

Reply:
We might want to find out the default value of `BOOST_THREAD_VERSION` then.

Reply:
It looks like `2` is the default version (https://www.boost.org/doc/libs/1_78_0/doc/html/thread/build.html#thread.build.configuration.version), and `5` the latest version, also including experimental features (https://www.boost.org/doc/libs/1_78_0/doc/html/thread/changes.html).

I don't want to downgrade the `BOOST_THREAD_VERSION`, so I'll undo the change.

Reply:
I agree.